### PR TITLE
[Wizard] Include .xtext file in jar created by gradle #284

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle [hunk 1]
```diff
@@ -15,6 +15,9 @@ jar {
 	from('model') {
 		into('model')
 	}
+	from(sourceSets.main.allSource) {
+		include '**/*.xtext'
+	}
```
Comment (spoenemann) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: Why is this necessary? If you include the xtext files in the resources, they'll be added to the jar automatically.

Comment (cdietrich) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: yes but in maven layout we have src/main/java with xtext+mwe2 and src/main/resources with nothing

Comment (spoenemann) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: Then it would be more correct to move the xtext file to src/main/resources. But ok, let's keep your solution so we don't change the current behavior too much.

Comment (cdietrich) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: yes i consider this as intended (xtext in stc main java)

Comment (cdietrich) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: is it ok to merge?

Comment (spoenemann) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.gradle/org.xtext.example.gradle.parent/gradle/source-layout.gradle]: 👍
